contracts/BSCValidatorSet.sol

```diff
@@ -813,6 +861,36 @@ contract BSCValidatorSet is IBSCValidatorSet, System, IParamSubscriber, IApplica
     emit validatorExitMaintenance(validator);
   }
 
+  function _accumulateRewardAndBlock(address valAddr, uint index, uint256 reward) private {
+
+    if (!sharingRewardInitialized) {
+      sharingRewardPercent = SHARING_REWARD_INIT_RATIO;
+      sharingRewardInitialized = true;
+    }
+
+    if (sharingRewardPercent > SHARING_REWARD_SCALE || SHARING_REWARD_SCALE == 0) {
```

Comment:
`SHARING_REWARD_SCALE == 0` seems unnecessary, it is a const rather than a var and it won't happen.